x-pack/plugins/alerting/server/rules_client/tests/create.test.ts

```diff
@@ -116,6 +116,19 @@ describe('create()', () => {
         isPreconfigured: false,
       },
     ]);
+    taskManager.schedule.mockResolvedValue({
+      id: 'task-123',
+      taskType: 'alerting:123',
+      scheduledAt: new Date(),
+      attempts: 1,
+      status: TaskStatus.Idle,
+      runAt: new Date(),
+      startedAt: null,
+      retryAt: null,
+      state: {},
+      params: {},
+      ownerId: null,
+    });
```

Comment:
This is a refactor to move repeated `taskManager.schedule.mockResolvedValue` code within `beforeEach` to offset the number of lines my tests add (trying to keep the file near 2000 lines).